recipes/dropstackframe/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "dropstackframe" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8f0b2724e729fee66d78ffaf15cfa1764e3a480f8c90f8dd6ed0e2b26c841c1a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - poetry-core
+    - python >=3.7
+    - pip
+  run:
+    - poetry
```

Comment:
Think this was only needed during the build. Not at runtime. Though please let me know if I'm missing something

```suggestion
```